source/client/ClickableImage.py

```diff
@@ -20,6 +20,10 @@ def __init__(self, image, x, y, width, height, outline_index):
 
     def draw(self, display, loc_xy, outline_choice):
         # Call this method to draw the ClickableImage on the screen,
+        #
+        # Future debug -- do I really need loc_xy in this?  Wrappedcard has a loc_xy already!
+        # Could replace this with new routine called clickableCard that uses fact that Image
```

Comment:
Ideally mark future work like this with TODO so that we can all find that sort of stuff with the same keyword search.